src/components/EventForm.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import "react-dates/initialize";
+import "react-dates/lib/css/_datepicker.css";
+import { DateRangePicker } from "react-dates";
+import moment from "moment";
+import uuid from "uuid";
+
+export default class EventForm extends React.Component {
+  constructor(props) {
```

Comment:
If you only want to initialize state, then you don't have to create `constructor`.